src/uxnrnbbs/hel2mos.cpp

```diff
@@ -49,18 +50,22 @@ int main(int argc, const char** argv) {
     }
     //	printf("started hel2mos\n");
 
-    char buf[256];
-    sprintf(buf, "%s/lib/helpdict", neuronhome());
-    String sf(buf);
-    InputFile* f = InputFile::open(sf);
-    if (f == nil) {
-        printf("Can't open %s\n", sf.string());
-        return 1;
+    {
```

Comment:
Seems like this whole file could be removed too?

Reply:
Maybe in another PR? (we are trying to keep it small)

Reply:
It is likely the whole folder is obsolete along with the mswin and mac versions. ```hel2mos``` dates back to 1993 and refers to being able to switch the gui to help mode so the cursor changes to a question mark and if you press on any gui element, a "Mosaic" browser window shows the relevant help page. But yes. let's do it separately.